css-typed-om/README.md

```diff
@@ -0,0 +1,116 @@
+# CSS Typed OM Explained
+
+CSS Typed OM is the extensible API for the CSSOM that reduces the burden of manipulating a CSS property's value via string manipulation. It does so by exposing CSS values as typed JavaScript objects rather than strings.
```

Comment:
nit: maybe s/the/an? It's probably not *the* extensible API.

Reply:
Done